libcontainer/cgroups/systemd/v2.go

```diff
@@ -307,9 +306,10 @@ func (m *unifiedManager) Destroy() error {
 		return err
 	}
 
-	// XXX this is probably not needed, systemd should handle it
-	err := os.Remove(m.path)
-	if err != nil && !os.IsNotExist(err) {
+	// systemd 239 do not remove sub-cgroups.
+	err := cgroups.RemovePath(m.path)
+	// cgroups.RemovePath has handled ErrNotExist
+	if err != nil {
```

Comment:
Hm.. didn't notice this when reviewing https://github.com/opencontainers/runc/pull/3226 but, given that this is the last code of the function, looks like we could just `return cgroups.RemovePath(m.path)` here.


No need to change, as this more closely matches what's in `main` (but we could clean that up)